src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedDeletedMethod.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Reflection;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.CodeGen;
+using Microsoft.CodeAnalysis.Symbols;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Represents a method that has been deleted in an Edit and Continue session, and whose body
+    /// throws a System.MissingMethodException.
+    /// </summary>
+    internal sealed class SynthesizedDeletedMethod : MethodSymbol, ISynthesizedDeletedMethod
```

Comment:
>SynthesizedDeletedMethod [](http://example.com/codeflow?start=26&length=24)

This might be a lot of work to create a full `MethodSymbol` that represents the deleted method. Is it necessary to have an actual `MethodSymbol`? If not, would it be easier to try generating a method definition in emit only?

Reply:
Yes, your point about the return type, parameters, etc. being from the old compilation is a very good one, and exactly the sort of comment I was looking for in putting this up early. Thank you. Maybe I went a little too simple in my "first simple scenario" :)

I will have a look and see if I can work out where to put some code in emit to do this, and expand my tests to validate the concerns too.